src/components/header/AppHeaderMenu.vue

```diff
@@ -0,0 +1,64 @@
+<script>
+export default {
+  data() {
+    return {
+      items: [
+        //  Change the names to the correct names
+        {
+          name: 'Features',
+          url: '/features'
+        },
+        {
+          name: 'Prices',
+          url: '/prices'
+        },
+        {
+          name: 'Community',
+          url: '/community'
+        },
+        {
+          name: 'Support',
+          url: '/support'
```

Comment:
Shall we put some event related names here for now, like Home, About, Speakers, Contact